src/Platform.Handlers/tests/Xamarin.Platform.Handlers.DeviceTests/Handlers/ProgressBar/ProgressBarTests.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Drawing;
+using System.Threading.Tasks;
+using Xamarin.Platform.Handlers.DeviceTests.Stubs;
+using Xunit;
+
+namespace Xamarin.Platform.Handlers.DeviceTests
+{
+	public partial class ProgressBarHandlerTests : HandlerTestBase<ProgressBarHandler>
+	{
+		[Fact]
+		public async Task ProgressInitializesCorrectly()
+		{
+			var progressBar = new ProgressBarStub()
+			{
+				Progress = 0.5
```

Comment:
Can we make this a theory with a few values? Like 0, 0.3, 0.5, 0.75 and 1? Maybe even add things out of the range, like -1, 32 and -0.45. This way we can test more than just what we know works.

Reply:
Changed the test to be a Theory with different values.